### PR TITLE
otherProperties Updates

### PR DESCRIPTION
**Pull request by erinspace**
Also change tags to subjects, as per the schema

This will require a migration (just like pubmed) --Fabian

### CODE REVIEW

**Comment by fabianvf** [comment]
Can you bring this up to date? I will then merge.

File: scrapi/base/helpers.py [hunk 1]
```diff
@@ -1,14 +1,42 @@
...
+from copy import deepcopy
 
+from pycountry import languages
 from nameparser import HumanName
 
+
 URL_REGEX = re.compile(ur'(https?://\S*\.\S*)')
 
 
+def build_properties(*args):
```
Comment (chrisseto) [inline scrapi/base/helpers.py]:
```python
[
    build_property(*args, **kwargs)
    for args, kwargs in 
    [
        pack(name, exp, desciption='foo'),
        pack(name, exp, uri='bar')
    ]
]
```

File: scrapi/base/helpers.py [hunk 2]
```diff
@@ -98,3 +126,10 @@ def oai_process_contributors(*args):
 
 def pack(*args, **kwargs):
     return args, kwargs
+
+
+def language_code(language):
+    try:
+        return languages.get(name=language)
```
Comment (chrisseto) [inline scrapi/base/helpers.py]: Does `.get` not have a defualt?

File: scrapi/base/transformer.py [hunk 3]
```diff
@@ -28,18 +29,24 @@ def _transform(self, schema, doc):
...
+            elif isinstance(value, tuple):
+                transformed[key] = self._transform_tuple(value, doc)
             elif isinstance(value, basestring):
                 transformed[key] = self._transform_string(value, doc)
             elif callable(value):
                 transformed[key] = value(doc)
         return transformed
 
-    def _transform_iterable(self, l, doc):
+    def _transform_list(self, l, doc):
```
Comment (chrisseto) [inline scrapi/base/transformer.py]: Does this really need to be a helper?

File: scrapi/base/helpers.py (hunk 2 above)
Comment (fabianvf) [inline scrapi/base/helpers.py]: no, it is pretty strange.

**Pull request merged**